pkg/kubelet/images/image_manager.go

```diff
@@ -85,7 +87,15 @@ func (m *imageManager) EnsureImageExists(pod *api.Pod, container *api.Container,
 		glog.Errorf("Couldn't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
 	}
 
-	spec := kubecontainer.ImageSpec{Image: container.Image}
+	// If the image contains no tag or digest, a default tag should be applied.
+	image, err := applyDefaultImageTag(container.Image)
```

Comment:
@Random-Liu @yujuhong To answer your question. `rkt fetch --no-store docker://busybox` has the same effect as `rkt fetch --no-store docker://busybox:latest`. So this PR should not affect rkt.